map/mwm_tests/mwm_foreach_test.cpp

```diff
@@ -22,14 +23,27 @@
 
 using namespace std;
 
-namespace
+namespace mwm_for_each_test
 {
 using Cont = vector<uint32_t>;
 
 bool IsDrawable(FeatureType & f, int scale)
 {
-  // Feature that doesn't have any geometry for m_scale returns empty DebugString().
-  return (!f.IsEmptyGeometry(scale) && feature::IsDrawableForIndex(f, scale));
+  if (f.IsEmptyGeometry(scale))
+    return false;
+
+  if (feature::IsDrawableForIndex(f, scale))
+    return true;
+
+  // Scale index ends on GetUpperScale(), but the actual visibility goes until GetUpperStyleScale().
+  if (scale != scales::GetUpperScale())
+    return false;
+
+  for (++scale; scale <= scales::GetUpperStyleScale(); ++scale)
```

Comment:
Разве это не эквивалентно?
```suggestion
  for (; ++scale <= scales::GetUpperStyleScale();)
```

Reply:
Эквивалентно, сокращу на while